resident-services/support-files/hierarchy_level.csv

```diff
@@ -1,9 +1,11 @@
 hierarchyLevel,name
 0,MyCountry
-1,Kenitra
-1,Rabat
 2,Kenitra
 2,Rabat
 3,Kenitra
 3,Rabat
-4,Rabat
+4,Mnasra
+4,Mograne
+4,Mehdia
+4,Agdal
+4,Souissi
```

Comment:
can we keep this also has a large file like with atleast 200 to 300 items ?

Reply:
Have updated the file with more values but 200-300 values are not there in the DB.